src/patient-search/patient-search-result/patient-search-result.component.css

```diff
@@ -0,0 +1,79 @@
+@media screen and (min-width: 769px) {
+  .patientResult {
+    display: flex;
+    justify-content: center;
+  }
+  .resultNumber {
+    display: block;
+    margin: 0rem 0.75rem;
+    color: var(--omrs-color-ink-medium-contrast);
+    padding: 1rem;
+  }
+  .patientCard {
+    border-radius: 0.75rem;
+  }
+  .tile {
+    padding: 0.625rem 1.5rem;
+  }
+}
+
+@media screen and (max-width: 769px) {
+  .resultNumber {
+    display: none;
+  }
+  .tile {
+    padding: 0.625rem 1rem;
+  }
+}
+.patientResult {
+  margin-top: 0.75rem;
+}
+.patientDetailsContainer {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.patientNameContainer {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 1rem;
+}
+
+.patientName {
+  font-size: 22px;
+  line-height: 129%;
+  letter-spacing: 0.03em;
+  font-weight: bold;
```

Comment:
bold can also be done with `omrs-bold`. The difference isn't significant, but the principle behind what i'm saying is just to think to use the styleguide as much as possible in our writing of css.

Reply:
fixed